common/crypto_utils/Makefile

```diff
@@ -12,11 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PY_VERSION=${shell python3 --version | sed 's/Python \(3\.[0-9]\).*/\1/' | tr -d .}
+#PY_VERSION=${shell python3 --version | sed 's/Python \(3\.[0-9]\).*/\1/' | tr -d .}
+PY_VERSION=${shell python3 --version | sed 's/Python \(3\.[0-9]\).*/\1/' | cut -b 1}
 MOD_VERSION=${shell ../../bin/get_version}
 
 # Format of wheel package is <pkg_name>-{python-tag}-{abi-tag}-{platform}
-WHEEL_FILE=dist/avalon_crypto_utils-${MOD_VERSION}-cp${PY_VERSION}-cp${PY_VERSION}m-linux_x86_64.whl
+WHEEL_FILE=dist/avalon_crypto_utils-${MOD_VERSION}-py${PY_VERSION}-none-any.whl
```

Comment:
Please remove below lines in `.clean` section
```
rm -f avalon_crypto_utils/crypto/crypto.py
rm -f avalon_crypto_utils/crypto/crypto_wrap.cpp
rm -f avalon_crypto_utils/verify_report/verify_report.py
rm -f avalon_crypto_utils/verify_report/verify_report_wrap.cpp
```

Reply:
Done